{
  try {
    processQueue();
  }
 catch (  Throwable t) {
    ExitUtil.terminate(1,getName() + " encountered fatal exception: " + t);
  }
}
